docs/files.md

```diff
@@ -2,16 +2,25 @@
 
 This chapter explains in detail how you manage and review files in Reviewable.  The file matrix provides an overview of the files in the pull request, while the individual file areas show diffs and line-specific discussions.
 
+<a id="revision-cells"></a>
 ## Revision cells
 
 A file's evolution over the course of the pull request is summarized in both the file matrix and the file areas by a row of revision cells.  Each cell represents one revision, with the leftmost ⊥ cell being a virtual base revision that changes to match the right diff bound.  Colors and icons in the revision cells summarize what happened in the pull request at each revision, as well as the latest review marks.
 
-![reviewable file matrix status](images/filematrix_9.png)
+![reviewable file matrix status](images/filematrix_10.png)
 
 The brackets in a row are the current left and right diff bounds for the file. To adjust the diff bounds, click on one desired revision bound and drag to the other one.  You can also just click on a revision to adjust the nearest bound.
 
 ![reviewable file matrix diff bounds](images/filematrix_8.png)
 
+By design, Reviewable doesn't track files that haven't deviated from their base form -- including files that have been modified back to their original state.  However, under certain circumstances (e.g., [rebasing](#rebasing)), Reviewable will "force" a final revision when it otherwise wouldn't.  This is indicated by a reverted icon in a kept cell and is designed to prevent you from missing a critical diff.  Reviewable will also add special styling to cells when [rebased heuristics](#heuristics) are in play.
+
+Symbol | Meaning
+:----: |---------
+![reviewable forced revision](images/filematrix_12.png) | File revision forced.
+![reviewable prior action](images/filematrix_11.png) | <strong>Inner Disc Color</strong>: File action against matched prior rebased revision. <br></br> <strong>Outer Background Color</strong>: File action against immediately preceding revision. <br></br> In this example, <i>r6</i> is not modified compared to <i>r4</i>, but is modified compared to <i>r5</i>.
```

Comment:
The image should include the revision arc above -- it's currently cropped out.  Otherwise, there's no context for why we'd care about comparing `r6` to `r4`.

Also, `br` is an odd element.  It's technically self-closing so you just need to use `<br>` by itself, but I typically use the XHTML variant `<br/`> instead as it's both compact and doesn't trigger my unbalanced tags reflex.